ega-data-api-res/src/main/resources/application.yml

```diff
@@ -1,7 +1,14 @@
-spring.profiles: no-oss
+spring.profiles: no-oss,LocalEGA,log-transfer,repo-logger,db-repo-logger
```

Comment:
This means that this file will be loaded with any of this profiles. If you want the other profiles to be loaded when the no-oss is used you have to use the spring.profiles.include

Reply:
ok, will try like that

Reply:
@omllobet for some reason that fails: using `spring.profiles.include` as suggested
see https://github.com/EGA-archive/ega-data-api/pull/132#discussion_r278181907

Reply:
I assume it fails because some profiles are not set up properly judging by this: https://github.com/EGA-archive/ega-data-api/blob/master/extras/docker-compose.yml#L27

Reply:
yes, as you said is probably is related to the db, maybe it's missing the lega functions or the schema it expects for the LocalEga context. So adding them to the spring.profiles.include makes them fail because of this but being on spring.profiles it does work because the log-transfer related profiles are not loaded

Reply:
Tried to fix it but I have no idea how, maybe @anandmohan777 can take a look when he has the time.

Reply:
Okay I will test it for these profiles

Reply:
@omllobet These profiles (`log-transfer`,`repo-logger`,`db-repo-logger`) requires a new set of schemas and tables and procedures. Could you provide me with that so that I can add them in DB init.sql file?

Reply:
@anandmohan777 https://github.com/EGA-archive/LocalEGA-db/tree/master/initdb.d the db schemas are there